tests/virtualization/virtman_storage.pm

```diff
@@ -75,7 +76,7 @@ sub create_nfs_share {
     send_key "ret";
     type_string "echo '$dir *(rw,sync)' >> /etc/exports";
     send_key "ret";
-    type_string "systemctl restart nfsserver.service";
+    systemctl 'restart nfsserver';
```

Comment:
Yeah, I know that it will work. I just see inconsistency for cups service we define exact name as it was before and here we have changed it for whatever reason, so just wanted a confirmation from @okurz that it's intentional.